scripts/.util/tools.json

```
@@ -0,0 +1,5 @@
+{
+  "jam": "v1.3.1",
+  "pack": "v0.26.0",
+  "syft": "v0.48.1"
```

nit, let's just use the latest versions from the get-go
```suggestion
  "jam": "v1.4.0",
  "pack": "v0.27.0",
  "syft": "v0.51.0"
```